pdns/rec-lua-conf.cc

```diff
@@ -389,6 +411,39 @@ void loadRecursorLuaConfig(const std::string& fname, luaConfigDelayedThreads& de
         lci.negAnchors.clear();
     });
 
+  Lua.writeFunction("readTrustAnchorsFromFile", [&lci](const std::string& fname) {
+      warnIfDNSSECDisabled("Warning: reading Trust Anchors from file (readTrustAnchorsFromFile), but dnssec is set to 'off'!");
+      auto zp = ZoneParserTNG(fname);
```

Comment:
Perhaps this should be moved inside the `try..` block so the message states that the error, if any, is about a `TA` file?